lodash.src.js

```diff
@@ -7325,25 +7325,17 @@
      * _.map(_.sortByAll(users, ['user', 'age']), _.values);
      * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
      */
-    function sortByAll() {
```

Comment:
I haven't had enough time to dedicate to naming issues like those. I'm
going to try to get it aliased at a minimum tho

On Sun, Apr 12, 2015 at 1:28 AM, John-David Dalton <notifications@github.com
> wrote:

> In lodash.src.js
> <https://github.com/lodash/lodash/pull/1125#discussion_r28200857>:
>
> > @@ -7325,25 +7325,17 @@
> >       * _.map(_.sortByAll(users, ['user', 'age']), _.values);
> >       * // => [['barney', 26], ['barney', 36], ['fred', 30], ['fred', 40]]
> >       */
> > -    function sortByAll() {
>
>  because of it's inclusion in ES6 but I'm glad regardless
>
> I'm not likely to take any more suggestions from you or Justin if the
> treatment I get is you all rolling over when presented to the Underscore
> crowd. Now you leave me in a shitty position of having to take their crap
> name or potentially causing compat issues if they work it into Backbone
> (which I believe is the end goal). All while Adam did his uninformed and
> inaccurate bemoaning of lodash in the Underscore issue that I'm unable to
> respond to.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lodash/lodash/pull/1125/files#r28200857>.
>